src/screens/login/Login.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { StatusBar, View } from 'react-native';
+import CheckBox from '@react-native-community/checkbox';
+import * as S from './styled';
+
+import AppButton from '../../components/AppButton';
+import InputField from '../../components/InputField';
+import { routes } from '../../constants/navigation/constants';
+
+function Login({ navigation }) {
+  return (
+    <S.SafeAreaView>
+      <StatusBar barStyle={'light-content'} backgroundColor={'#222222'} />
+      <S.ContentView>
+        <View>
+          <S.TitleText>
+            Entrar <S.HighlightText>M-PAY</S.HighlightText>
+          </S.TitleText>
+        </View>
+        <View>
+          <S.MarginView>
```

Comment:
⛏️ Esse componente não ajuda a entender o que ele faz, além de deixar implícito que tem uma margem. Como está sendo usado sempre como container do input, pode ser nomeado como tal (e nesse cenário, o espaçamento estar implícito não é um problema).
```suggestion
          <S.TextInputContainer>
```
Também deixaria explícito que é um TextInput, por que podemos ter inputs específicos para outros tipos de dados como número, senha, URL, etc. E especialização geralmente escala melhor do que super generalização (um componente ser usado para vários cenários).

Reply:
👍